Library/OcCryptoLib/RsaDigitalSign.c

```diff
@@ -0,0 +1,1047 @@
+/** @file
+
+OcCryptoLib
+
+Copyright (c) 2018, savvas
+
+All rights reserved.
+
+This program and the accompanying materials
+are licensed and made available under the terms and conditions of the BSD License
+which accompanies this distribution.  The full text of the license may be found at
+http://opensource.org/licenses/bsd-license.php
+
+THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+/**
+  Copyright (c) 2014 The Chromium OS Authors. All rights reserved.
+  Use of this source code is governed by a BSD-style license that can be
+  found in the LICENSE file.
+
+  Implementation of RSA signature verification which uses a pre-processed key
+  for computation.
+**/
+
+#ifdef EFIAPI
+#include <Library/BaseMemoryLib.h>
+#include <Library/DebugLib.h>
+#include <Library/BaseLib.h>
+#include <Library/OcGuardLib.h>
+#endif
+
+#include <Library/OcCryptoLib.h>
+
+//
+// RSA algorithm paddings
+//
+STATIC CONST UINT8 PADDING_RSA2048_SHA256[CONFIG_RSA2048_NUM_BYTES - SHA256_DIGEST_SIZE] = {
```

Comment:
Hrm, maybe it is better to follow suit and not to name non-macros and non-types in capital snake case? Could you please change the names to mPaddingRsa2048Sha256, just as it is everywhere else in the codebase?

Reply:
Done. https://github.com/acidanthera/OcSupportPkg/commit/53bee39c816b29514d8bdb17369ee6d2906c2c92